pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -444,21 +445,49 @@ func (pc *PodController) configurePodSecondaryNetwork(pod *corev1.Pod, networkLi
 		}
 
 		// Secondary network information retrieved from API server. Proceed to configure secondary interface now.
-		if err = pc.configureSecondaryInterface(pod, network, resourceName, podCNIInfo, networkConfig); err != nil {
+		res, err := pc.configureSecondaryInterface(pod, network, resourceName, podCNIInfo, networkConfig)
+		if err != nil {
 			klog.ErrorS(err, "Secondary interface configuration failed",
 				"Pod", klog.KRef(pod.Namespace, pod.Name), "interface", network.InterfaceRequest,
 				"networkType", networkConfig.NetworkType)
 			savedErr = err
 		} else {
 			interfacesConfigured++
 		}
+		if res != nil {
+			status := &netdefv1.NetworkStatus{}
+			status.Name = network.Name
+			status.Default = false
```

Comment:
what does this Default=false mean?

Reply:
it seems multus also includes the primary network interface in the annotation?